### PR TITLE
make more snotes L_NETWIDE

### PR DESCRIPTION
**Pull request by jesopo**
this switches a lot of important snotes from local-server-only targets (`L_ALL`, `L_ADMIN`) to network-wide (`L_NETWIDE`), because most modern networks are centrally-operated and having to have connections on all leaves to read important snotes isn't great.

pulled out of chary PR, hope the squash merge between upstreams worked ok

### CODE REVIEW

**Review by edk0 — CHANGES_REQUESTED** [review]
Thanks for this. I think m_dline probably shouldn't be skipped—D-lines are (currently) local, there should be one netwide snote for every server that gets them.

**Comment by jesopo** [comment]
some comments I forgot to bring forward from the chary PR:

I skipped over m_kill because I couldn't understand some seemingly important things, and I skipped over m_dline because the ircd-seven changes above didn't make sense (netwide snote per server?)

some changes (m_kline, m_resv, m_squit, m_xline) were basically ripped straight out of the ircd-seven changeset without me fully grokking them, because I got a bit lost in how propagation works for those

**Comment by edk0** [comment]
Ack, but I'm pretty sure the seven changes do make sense for dline. They don't all have to be `ON *`.

**Review by edk0 — APPROVED** [review]

File: ircd/client.c [hunk 1]
```diff
@@ -2135,7 +2135,7 @@ error_exit_client(struct Client *client_p, int error)
 	{
 		if(error == 0)
 		{
-			sendto_realops_snomask(SNO_GENERAL, is_remote_connect(client_p) && !IsServer(client_p) ? L_NETWIDE : L_ALL,
+			sendto_realops_snomask(SNO_GENERAL, !IsServer(client_p) ? L_NETWIDE : L_ALL,
```
Comment (dwfreed) [inline ircd/client.c:2138]:
```suggestion
			sendto_realops_snomask(SNO_GENERAL, L_NETWIDE,
```

File: ircd/client.c [hunk 2]
```diff
@@ -2144,7 +2144,7 @@ error_exit_client(struct Client *client_p, int error)
 		}
 		else
 		{
-			sendto_realops_snomask(SNO_GENERAL, is_remote_connect(client_p) && !IsServer(client_p) ? L_NETWIDE : L_ALL,
+			sendto_realops_snomask(SNO_GENERAL, !IsServer(client_p) ? L_NETWIDE : L_ALL,
```
Comment (dwfreed) [inline ircd/client.c:2147]:
```suggestion
			sendto_realops_snomask(SNO_GENERAL, L_NETWIDE,
```

File: ircd/sslproc.c [hunk 3]
```diff
@@ -427,7 +427,7 @@ ssl_process_dead_fd(ssl_ctl_t * ctl, ssl_ctl_buf_t * ctl_buf)
 
 	if(IsAnyServer(client_p))
 	{
-		sendto_realops_snomask(SNO_GENERAL, is_remote_connect(client_p) && !IsServer(client_p) ? L_NETWIDE : L_ALL, "ssld error for %s: %s", client_p->name, reason);
+		sendto_realops_snomask(SNO_GENERAL, !IsServer(client_p) ? L_NETWIDE : L_ALL, "ssld error for %s: %s", client_p->name, reason);
```
Comment (dwfreed) [inline ircd/sslproc.c:430]:
```suggestion
		sendto_realops_snomask(SNO_GENERAL, L_NETWIDE, "ssld error for %s: %s", client_p->name, reason);
```

**Review by dwfreed — CHANGES_REQUESTED** [review]
A few that should always be netwide

**Review by dwfreed — APPROVED** [review]

**Pull request merged**